src/components/WorkOrder/Update/index.js

```diff
@@ -9,14 +9,15 @@ import { isSpendLimitReachedResponse } from '@/utils/helpers/apiResponses'
 import WorkOrderUpdateForm from './Form'
 import WorkOrderUpdateSummary from './Summary'
 import WorkOrderUpdateSuccess from './Success'
+import { PURDY_CONTRACTOR_REFERENCE } from '@/utils/constants'
 
 const WorkOrderUpdateView = ({ reference }) => {
   const [loading, setLoading] = useState(false)
   const [error, setError] = useState()
   const [currentUser, setCurrentUser] = useState({})
   const [tasks, setTasks] = useState([])
   const [originalTasks, setOriginalTasks] = useState([])
-  const [sorCodes, setSorCodes] = useState([])
+  const [workOrder, setWorkOrder] = useState([])
```

Comment:
Is there a particular reason why we need to use `array[]` instead of `object{}`?
Cause from b/e WorkOrder comes as an object
`{reference: 10000954, dateRaised: '2022-04-21T10:54:48.394808Z',.....}`

Reply:
No :) good spot - updated to use `useState()`